tools/depmod.c

```diff
@@ -227,7 +227,10 @@ static int index_add_value(struct index_value **values,
 		values = &(*values)->next;
 
 	len = strlen(value);
-	v = NOFAIL(calloc(1, sizeof(struct index_value) + len + 1));
+	v = calloc(1, sizeof(struct index_value) + len + 1);
+	if (v == NULL)
+		return -1;
+
```

Comment:
It would split allocation and assignment of values within `v`. Since we do not actually modify the content where the pointer points to but the pointer itself, i.e. a stack variable only visible to the function, "this is fine."